src/internal/coredb/projects.go

```diff
@@ -0,0 +1,88 @@
+package coredb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/gogo/protobuf/types"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+	"time"
+)
+
+// Querier defines an interface for functions shared across sqlx.Tx and sqlx.DB types.
+// Functions that take a querier support both running inside a transaction and outside it.
+type Querier interface {
```

Comment:
Can we just embed the sqlx interfaces? Like
```
type Querier interface {
    sqlx.ExecerContext
    sqlx.QueryerContext
}
```

Reply:
sure, I can try that